core/src/main/scala/org/apache/spark/eventhubs/EventHubsConf.scala

```diff
@@ -436,6 +437,18 @@ final class EventHubsConf private (private val connectionStr: String)
     set(MaxEventsPerTriggerKey, limit)
   }
 
+  def setMetricPlugin(metricPlugin: MetricPlugin): EventHubsConf = {
```

Comment:
You also need to add "eventhubs.metricPlugin" to trimmed version of EventHubsConf - line 162.